src/ripple/rpc/handlers/Catalogue.cpp

```diff
@@ -72,6 +72,25 @@ static constexpr uint16_t CATALOGUE_COMPRESS_LEVEL_MASK =
 static constexpr uint16_t CATALOGUE_RESERVED_MASK =
     0xF000;  // Bits 12-15: reserved
 
+std::string
+formatFileSize(uint64_t bytes)
+{
+    static const char* units[] = {"B", "KB", "MB", "GB", "TB", "PB"};
```

Comment:
<img width="295" alt="image" src="https://github.com/user-attachments/assets/40140ded-b9e6-4deb-acc3-4d815ca27a91" />
Best practice to use the IEC standard imho... KiB etc...